apps/api/src/app/notification-groups/notification-groups.controller.ts

```diff
@@ -44,6 +64,30 @@ export class NotificationGroupsController {
     );
   }
 
+  @Put('/:id')
+  @ExternalApiAccessible()
+  @ApiCreatedResponse({
+    type: NotificationGroupResponseDto,
+  })
+  @ApiOperation({
+    summary: 'Update notification group',
+  })
+  updateNotificationGroup(
+    @UserSession() user: IJwtPayload,
+    @Param('id') id: string,
+    @Body() body: CreateNotificationGroupRequestDto
+  ): Promise<NotificationGroupResponseDto> {
```

Comment:
You are using a PUT HTTP method in the endpoint definition but you have a PATCH method implementation as you only send the fields of the entity you want to change.
So I would suggest you to either one of both:
- you change the implementation of this endpoint to expect and take the full entity to be updated in the database, or
- you modify the method of the endpoint to PATCH and leave the current implementation your did.